app/mailers/event_mailer.rb

```diff
@@ -24,8 +24,31 @@ def occurrence(occurence, addresses)
                                        title: @event.title, times: @occurrence_count))
   end
 
+  def assign(event, assigner)
+    @event = event
+    @link = link
+    @assignee = event.user
+    @assigner = assigner
+    @render_assignee_analytics = render_assignee_analytics?
+    add_assigner_photo
+    mail(to: @assignee.email,
+         subject: default_i18n_subject(project_name: @event.project.name,
+                                       environment: @event.environment,
+                                       title: @event.title, assigner_name: @assigner.nickname))
+  end
+
   private
 
+  def render_assignee_analytics?
+    @event.occurrence_count.positive? || @event.occurrences_today? || @assignee.assigned_in_project?(@event.project_id)
```

Comment:
all these query methods reused in partial, should we consider putting them into instance variables?
@Alexey-Lukin